### PR TITLE
Fix issue #17 - IE11 page hangs on empty input triggering

### PR DESCRIPTION
**Pull request by joe-campiondigital-mo**
This fixes https://github.com/jondavidjohn/payform/issues/17

IE11 set focus to an empty field if focus was lost. When focus was moved to another empty field this would cause a loop and melt the browser. There are no negatives fixing it like this as far as I can see as we don't want to format `""` values anyway.

### CODE REVIEW

**Comment by joe-campiondigital-mo** [comment]
Hmm.. this doesn't fix the minified jQuery build. It works for the un-minified version.

I might have to re-visit this.

**Comment by joe-campiondigital-mo** [comment]
I had to tweak the build script to use uglifyjs rather than uglify as it was changing [this](https://github.com/jondavidjohn/payform/pull/32/files#diff-e17bc2209f2f1c09650b6ae8d6848b62R319) logic. `===` was being converted to `!==` on minification. Very strange!

File: src/payform.coffee [hunk 1]
```diff
@@ -310,6 +314,8 @@
   # Format CVC
 
   reFormatCVC = (e) ->
+    if e.target.value is ""
+      return
```
Comment (jondavidjohn) [inline src/payform.coffee]: These could be shortened to be `return if e.target.value is ""`

**Comment by jondavidjohn** [comment]
@hppycoder Have you verified this does in fact fix the issue for IE11?

**Comment by hppycoder** [comment]
I will verify Tuesday. We are looking at changing our code out and our qa will be happy to look.

**Comment by hppycoder** [comment]
I am sorry it's taken me so long to test this. I am confirm that this code change will 100% fix the IE11 issue. I tested this by having a virtualbox with IE11 and Windows10 on it. I ensured that the current working example of: https://jondavidjohn.github.io/payform/ failed.

Not only did it lock the browser up but it did it with style! It moved the cursor between fields over and over and finally crashed the browser.

I then brought the example into a plnkr. I used the origional "payform.min.js" from the site and tested my plnkr. Found that it had the same affect as the current site to verify my plnkr was setup correctly. Then took the modified "payform.min.js" from the PR and tried again. It's working smoothly and does not crash the browser.

Please merge this and create a release, I would like to bring into my current project as soon as possible.

**Comment by jondavidjohn** [comment]
Thanks for testing @hppycoder !

**Pull request merged**